native/opencv-gpu.cc

```diff
@@ -1,16 +1,15 @@
+#include <opencv2/cudaobjdetect.hpp>
+
 #include "opencv-gpu.h"
 #include "opencv-wrapper.h"
 #include "utils.h"
-#include <opencv2/cudaobjdetect.hpp>
 
 EXTERN_C_BEGIN
 
 // =============================================================================
 //   Basic
 // =============================================================================
-void* cv_gpu_mat_default() {
-    return new cv::cuda::GpuMat();
-}
+void* cv_gpu_mat_default() { return new cv::cuda::GpuMat(); }
```

Comment:
Probably we want always use multiline functions?

Reply:
Yes, I am tweaking the formatting rules now... Before committing new changes, I want to see Travis is properly configured.